apps/designer/app/canvas/shared/use-designer-ready.ts

```diff
@@ -1,18 +1,30 @@
 import { useState } from "react";
+import { useInterval } from "react-use";
 import { publish, useSubscribe } from "~/shared/pubsub";
 
 declare module "~/shared/pubsub" {
   export interface PubsubMap {
-    designerReady: undefined;
-    designerReadyAck: undefined;
+    canvasReady: undefined;
+    canvasReadyAck: undefined;
   }
 }
 
 export const useSubscribeDesignerReady = () => {
```

Comment:
would make sense to also rename the hook and the file, no?

Reply:
I did that at first, but then realised it kinda still checks that designer is ready, but now by receiving `canvasReadyAck` instead of `designerReady`